src/test/java/com/wikia/webdriver/testcases/discussions/ZeroErrorStateTests.java

```diff
@@ -32,43 +31,46 @@
   private static final String FOLLOW_MESSAGE_HEADER_TEXT = "Welcome to your Following tab.";
   private static final String FOLLOW_MESSAGE_CONTENT_TEXT = "Hit the “Follow” icon at the bottom of any post to fill your list with discussions that matter most to you. We’ll put them here and notify you of new activity.";
   private static final String FOLLOW_MESSAGE_BUTTON_TEXT = "FIND POSTS TO FOLLOW";
-  public static final String NO_REPLIES_ICON_MESSAGE = "No replies icon should be visible.";
-  public static final String NO_REPLIES_UNDER_POST_MESSAGE = "There should be no replies on new post (without replies).";
+  private static final String NO_REPLIES_ICON_MESSAGE = "No replies icon should be visible.";
+  private static final String NO_REPLIES_UNDER_POST_MESSAGE = "There should be no replies on new post (without replies).";
+
+  private static final String DESKTOP = "discussions-zero-error-state-desktop";
+  private static final String MOBILE = "discussions-zero-error-state-mobile";
 
   /**
    * ANONS ON DESKTOP SECTION
    */
 
-  @Test(enabled = false, groups = "discussions-anonUserOnDesktopSeesProperMessageWhenOpensEmptyReportedPostsPage")
+  @Test(groups = DESKTOP, enabled = false)
   @Execute(asUser = User.ANONYMOUS)
   @InBrowser(emulator = Emulator.DESKTOP_BREAKPOINT_BIG)
   @RelatedIssue(issueID = "SOC-3667")
```

Comment:
Just an suggestion to keep `@Test(enabled=false)` closer to `@RelatedIssue`.

Reply:
:+1: